docs/Agent/installation/ansible_based_installation.md

```diff
@@ -1,20 +1,16 @@
 # Ansible based installation
 
-In the following, we describe how to install `pbench-agent` using an ansible playbook.
+In the following, we describe how to install Pbench Agent` using an ansible playbook.
 
 :::{note}
-The same pbench-agent version must be installed on all the test systems that participate in a benchmark run, there is no support for mixed installations.
+The same Pbench Agent version must be installed on all the test systems that participate in a benchmark run, there is no support for mixed installations.
 :::
 
 ## Setup
 
-1. Make sure that you have ansible and ansible-galaxy installed on your laptop (or wherever you decide to run the playbooks).
+1. Make sure that you have `ansible` package installed.
```

Comment:
"the ansible package". Do we need the code font here? You didn't have it before, and I'm not sure it's useful.

Reply:
fixed